server/src/main/java/org/opensearch/action/admin/cluster/node/stats/NodeStats.java

```diff
@@ -128,6 +129,9 @@ public class NodeStats extends BaseNodeResponse implements ToXContentFragment {
     @Nullable
     private SearchBackpressureStats searchBackpressureStats;
 
+    @Nullable
+    private SegmentReplicationStats segmentReplicationStats;
```

Comment:
We are only displaying the back pressure rejection count here and don't need to compute everything this object has. Let's make a new segmentreplicationRejectionStats object instead? We can return that right from the tracker.

Reply:
Added in new commit